frontend/src/app/pages/marker-page/marker-page.component.ts

```diff
@@ -124,6 +124,12 @@ export class MarkerPageComponent implements OnInit {
     }
 
     public skipScan(): void {
+        this.scanService.skipScan(this.scan.scanId).then(() => {
```

Comment:
What if this request will be rejected? I think we should move to the next Scan anyway. 

I would send `skipScan` and `nextScan` concurrently and not wait for skip request to be resolved/rejected:
```js
public skipScan(): void {
    this.scanService.skipScan(this.scan.scanId);
    this.nextScan();
}
```

It will speed up loading new Scan ~2x :)